src/backend/cuda/nvrtc/cache.cpp

```diff
@@ -120,6 +145,8 @@ Kernel buildKernel(const int device, const string &nameExpr,
     } else {
         constexpr static const char *includeNames[] = {
             "math.h",          // DUMMY ENTRY TO SATISFY cuComplex_h inclusion
+            "stdbool.h",       // DUMMY ENTRY TO SATISFY af/defines.h inclusion
```

Comment:
I thought I removed these via ifdefs.

Reply:
Nope, the problem was still there before this fix. Also, we shouldn't add internal specific guards to `af/defines.h` header.

Reply:
Its a compiler guard similar to checking for gcc or clang. It is not defined by ArrayFire.